### PR TITLE
Add availability help flyout

### PR DESCRIPTION
**Pull request by eugenesk24**
### Description
Add availability help flyout to availability configuration panel

### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: dashboards-observability/public/components/application_analytics/components/flyout_components/availability_info_flyout.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+            <li>color: The color of the availability badge on the home page</li>
+            <li>name: The text in the availability badge on the home page</li>
+            <li>expression: Comparison operator to determine the availability</li>
+            <li>value: Value to use when calculating availability</li>
+          </ul>
+          <h3>Configuring availability</h3>
+          By default, Application analytics shows results from the last 24 hours of your data. To
+          see data from a different timeframe, use the date and time selector.
+          <h3>Time series metric</h3>A time series metric is any visualization that has a query that
+          spans over a timestamp and is a bar/line chart. Users can use the power of the PPL
```
Comment (Naarcha-AWS) [inline dashboards-observability/public/components/application_analytics/components/flyout_components/availability_info_flyout.tsx:47]: Switch second sentence on line 47 to, "You can use the PPL language to define arbitrary conditions on their logs to create a visualization over time."

**Review by Naarcha-AWS — APPROVED** [review]
One small note, but otherwise LGTM.

File: dashboards-observability/public/components/application_analytics/components/flyout_components/availability_info_flyout.tsx (hunk 1 above)
Comment (eugenesk24) [reply dashboards-observability/public/components/application_analytics/components/flyout_components/availability_info_flyout.tsx:47]: Should it be *your* logs?

Comment (Naarcha-AWS) [reply dashboards-observability/public/components/application_analytics/components/flyout_components/availability_info_flyout.tsx:47]: Correct, good catch :D

**Review by Naarcha-AWS — APPROVED** [review]

File: dashboards-observability/public/components/event_analytics/explorer/visualizations/config_panel/config_panes/config_controls/config_availability.tsx [hunk 2]
```diff
@@ -50,6 +54,13 @@ export const ConfigAvailability = ({ visualizations, onConfigChange, vizState =
     { value: '≠', text: '≠' },
   ];
 
+  const availabilityAccordionButton = (
+    <EuiFlexGroup direction="row" justifyContent="center">
+      <EuiText>&nbsp;&nbsp;Availability&nbsp;</EuiText>
```
Comment (ps48) [inline dashboards-observability/public/components/event_analytics/explorer/visualizations/config_panel/config_panes/config_controls/config_availability.tsx:59]: You should try using padding/margin here.

**Review by ps48 — APPROVED** [review]

**Review by joshuali925 — APPROVED** [review]

**Pull request merged**